src/App.js

```diff
@@ -1,10 +1,50 @@
-import React from 'react';
+import {useState, useEffect} from 'react';
 import './App.css';
+import Ditto from './Ditto'
+import axios from "axios";
+
 
 function App() {
+    const [pokenames, setPokenames] = useState(null);
+    const [limit, setLimit] = useState(20);
+
+
+
+    useEffect(() => {
+        console.log("fetchnames")
+
+        async  function fetchNames(){
+            const response = await axios.get(`https://pokeapi.co/api/v2/pokemon?limit=${limit}`);
+            console.log(response.data.results)
+
+            setPokenames(response.data.results);
+
+        }
+
+        fetchNames()
+
+
+
+
+    }, [])
+
+
+    console.log(pokenames)
+
   return (
+
     <div>
-      Begin hier met de opdracht!
+
+        {pokenames ? <div>
+            {/*<Ditto nameOfPokemon={"jigglypuff"}/>*/}
+            {/*<Ditto nameOfPokemon={"diglett"}/>*/}
+            {/*<Ditto nameOfPokemon={"bulbasaur"}/>*/}
+            {/*<Ditto nameOfPokemon={"pikachu"}/>*/}
+            {/*<Ditto nameOfPokemon={"scyther"}/>*/}
+            {pokenames.map(pokemon => {
+                return <Ditto nameOfPokemon={pokemon.name}/>
+            })}
+        </div> : <h3>Loading Pokenames for you my friend</h3> }
```

Comment:
Als er initieel een error is, dan blijft deze altijd zo staan. Het liefst zou ik in de useEffect willen zien dat een state `loading` op true wordt gezet. Dan wordt het wat meer expliciet